src/lib/events.js

```diff
@@ -32,6 +32,7 @@ export const ERROR_CODE = {
     GENERIC: 'error_generic',
     IMAGE_SIZING: 'error_image_sizing',
     INVALID_CACHE_ATTEMPT: 'error_invalid_file_for_cache',
+    LARGE_FILE_SIZE: 'error_large_size_file',
```

Comment:
Is the issue file size or representation size? If the latter, should this be `CONVERSION_TOO_LARGE` or something?

Reply:
This is an issue with the size of the original file. If it is larger than a certain size, a conversion to a PDF representation will not be attempted.

Reply:
I will add the prefix, I think that makes sense.